questionnaire.js

```diff
@@ -1093,14 +1093,15 @@ export function displayQuestion(nextElement) {
   );
 
   // check all responses for next question
-  [...nextElement.children]
+  [...(nextElement.querySelector('fieldset')?.children || [])]
```

Comment:
Can we change this to:

nextElement.querySelectorAll('[displayif]')
   .map...